### DISCUSSION TITLE
Valoracion practica

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Sensores-y-Actuadores/practica-de-la-1er-semana-grupo-3#56] [Announcements]
Nota Grupal: 7(Siete)

Carla Argentina Wayar: Silla Inteligente. (No realiza diagrama en bloque interno): 7 (Siete)
Dario Arriola; Monitor de estado del automovil (No realiza diagrama en bloque int): 6 (Seis)
Gonzalo pasarell; Sommier Inteligente (No realiza diagrama interno): 7 (Siete)
Lionel Rios; Kayak Inteligente (NO realiza diagrama interno): 7 (siete)
Oscar Gazzola; Supervision y control de termotanque IoT : 7 (Siete)
Jeremias Castro; Riego Automatizado. (No realiza diagrama en bloque interno): 6 (Seis)

**Comment by WayarCarla** [comment]
BUENAS PROFESOR LA ACTIVIDAD NO TERMINABA HOY VIERNES ? NO COMPRENDO SERIA BUENO QUE COLOCARA BIEN LA FECHA ASI NO TENEMOS TAREAS INCOMPLETAS. GRACIAS
